test/acceptance/features/bindAppToService.feature

```diff
@@ -217,3 +217,177 @@ Feature: Bind an application to a service
+                              displayName: SVC name
+                              path: svcName
+                              x-descriptors:
+                                - binding:env:attribute
+
+            """
+        * The CRD "backservs.service.example.com" is present
+            """
+            apiVersion: apiextensions.k8s.io/v1beta1
+            kind: CustomResourceDefinition
+            metadata:
+                name: backservs.service.example.com
+            spec:
+                group: service.example.com
+                versions:
+                    - name: v1
+                      served: true
+                      storage: true
+                scope: Namespaced
+                names:
+                    plural: backservs
+                    singular: backserv
+                    kind: Backserv
+                    shortNames:
+                    - bs
+            """
+        * The CR "demo-backserv-cr-1" for kind "Backserv" is present
+            """
+            apiVersion: service.example.com/v1
+            kind: Backserv
+            metadata:
+                name: demo-backserv-cr-1
+            spec:
+                svcName: demo-backserv-cr-1
+            """
+        When Service Binding Request is applied to connect the database and the application
+            """
+            apiVersion: apps.openshift.io/v1alpha1
+            kind: ServiceBindingRequest
+            metadata:
+                name: sbr-csv-attribute
+            spec:
+                backingServiceSelector:
+                    group: service.example.com
+                    version: v1
+                    kind: Backserv
+                    resourceRef: demo-backserv-cr-1
+            """
+        Then Secret "sbr-csv-attribute" must contain data from resource type "Backserv" cr name "demo-backserv-cr-1" and jsonpath "{.spec.svcName}"
+        And jq ".status.conditions[] | select(.type=="CollectionReady").status" of Service Binding Request "sbr-csv-attribute" should be changed to "True"
+        And jq ".status.conditions[] | select(.type=="InjectionReady").status" of Service Binding Request "sbr-csv-attribute" should be changed to "False"
+
+    Scenario: Create intermidiate secret using information mentioned in x-descriptors of backend service CSV
+        Given Backend service "some-backend-service.v0.2.0" CSV is installed
+            """
+            apiVersion: operators.coreos.com/v1alpha1
+            kind: ClusterServiceVersion
+            metadata:
+                name: some-backend-service.v0.2.0
+            spec:
+                displayName: Some Backend Service
+                install:
+                    strategy: deployment
+                customresourcedefinitions:
+                    owned:
+                        - name: backservs.service.example.com
+                          version: v1
+                          kind: Backserv
+                          statusDescriptors:
```

Comment:
there is also `specDescriptors` - we should also have a test case covering that  as well.

Reply:
SpecDescriptors is covered in the above scenario. https://github.com/redhat-developer/service-binding-operator/pull/620/files#diff-cccd4debaac09aaed0262fe2dbbad9e1R238

Reply:
ok